.github/workflows/main.yml

```diff
@@ -19,16 +19,29 @@ jobs:
     # conrainer with pre-installed mender-artifact and mender-cli tools
     container:
       image: mendersoftware/mender-ci-tools:mender-master
-    name: Upload a dummy Mender artifact
+    name: Deploy a dummy Mender Artifact
     steps:
       # Checkout git repository
       - uses: actions/checkout@v3
       # Download Mender bash client
+      - name: Serup environment
+        run: |
+          apk add openssl jq
+          # secrets are not passed to the runner when a workflow is triggered from a forked repository
+          # https://docs.github.com/en/actions/security-guides/encrypted-secrets#using-encrypted-secrets-in-a-workflow
+          if [[ -z "${{ secrets.MENDER_TENANT_TOKEN }}" || "${{ secrets.MENDER_TENANT_TOKEN }}" == "" ]]; then
+            echo "ERROR: MENDER_TENANT_TOKEN is empty."
+            exit 1
+          fi
+          if [[ -z "${{ secrets.MENDER_SERVER_ACCESS_TOKEN }}" || "${{ secrets.MENDER_SERVER_ACCESS_TOKEN }}" == "" ]]; then
+            echo "ERROR: MENDER_SERVER_ACCESS_TOKEN is empty."
+            exit 1
+          fi
       - name: Setup Mender bash client
         run: |
-          curl -O https://raw.githubusercontent.com/mendersoftware/bash-mender-client/master/mender-client.sh
+          # is needed while latest changes in not cherry picked to mendersoftware/bash-mender-client
+          curl -O https://raw.githubusercontent.com/mirzak/bash-mender-client/d5a4ded71404f8526362f5b94e7f79065cfced09/mender-client.sh
```

Comment:
Let's merge this one first, then :+1: https://github.com/mirzak/bash-mender-client/pull/1

(We cannot point here to a commit that does not belong to any branch)

Reply:
I don't think we can merge it there and doesn't look like Mirza is managing it as well. If it's not the way to go with pointing to a commit, then let me cherry pick latest changes from `saas-tools` to [mendersoftware/bash-mender-client](https://github.com/mendersoftware/bash-mender-client)

Reply:
> (...), then let me cherry pick latest changes from `saas-tools` to [mendersoftware/bash-mender-client](https://github.com/mendersoftware/bash-mender-client)

Exactly. We need to update our fork and launch your PR there.

Reply:
@lluiscampos what do you think about adding bash client to the `mender-ci-tools` image?

Reply:
I don't think it makes sense. It is not a CI tool, it is only for testing. Not to mention that it is far from production quality.

It makes sense to have it's own Docker image though, as we have today from `saas-tools`, and use it from other places. But we can follow-up on that separately.

Reply:
1st PR: https://github.com/mendersoftware/saas-tools/pull/202

Reply:
2nd PR: https://github.com/mendersoftware/bash-mender-client/pull/1